object/service.proto

```diff
@@ -295,15 +295,48 @@ message SearchRequest {
 
     // Version of the Query Language used
     uint32 version = 2;
-    // Filter structure
+    // Filter structure checks if object header field or attribute content
+    // matches a value.
+    //
+    // By default `name` field refers to the corresponding object's `Attribute`.
+    // Some Object's header fields can also be accessed by adding `$Object:`
+    // prefix to the name. Here is the list of fields available via this prefix:
+    //
+    // * $Object:version \
+    //   version
+    // * $Object:containerID \
+    //   container_id
+    // * $Object:ownerID \
+    //   owner_id
+    // * $Object:creationEpoch \
+    //   creation_epoch
+    // * $Object:payloadLength \
+    //   payload_length
+    // * $Object:payloadHash \
+    //   payload_hash
+    // * $Object:objectType \
+    //   object_type
+    // * $Object:homomorphicHash \
+    //   homomorphic_hash
+    //
+    // There are some well-known filter aliases to match objects by certain propeties:
```

Comment:
I think it would be useful to additionally describe the way of specifying filter values for this class.

Reply:
To be more specific. Now it implemented this way:
- `$Object:ROOT` key with `true` value returns only root objects, with any other value returns non root object
- `$Object:LEAF` key with `true` value returns only leaf objects, with any other value returns non leaf object
- `$Object:PARENT` key with **any** value returns objects with children list in split header (zero-object?)
- `$Object:CHILDFREE` with **any** value returns objects without children list in split header (all non-zero objects). 

Maybe we should merge last two to `CHILDFREE` with `true` and any other value.

Reply:
Agree. Will fix.